Gruntfile.js

```diff
@@ -148,7 +148,7 @@ module.exports = function (grunt) {
 
     exec: {
       appOpen: {
-        cmd: `~/Applications/Google\\ Chrome.app/Contents/MacOS/Google\\ Chrome --load-and-launch-app="${pluginFolder}"`
+        cmd: `/Applications/Google\\ Chrome.app/Contents/MacOS/Google\\ Chrome --load-and-launch-app="${pluginFolder}"`
```

Comment:
Intentional to change this path?

Reply:
Yep! Sorry, I should've pulled it out to a separate commit/PR.

Most folks seem to have Chrome installed for all users (`/Applications`) vs. just the current user (`~/Applications`).